test/ReactWrapper-spec.js

```diff
@@ -1896,4 +1896,62 @@ describeWithDOM('mount', () => {
       expect(wrapper.at(1).key()).to.equal('bar');
     });
   });
+
+  describe('.name()', () => {
+    describe('node with displayName', () => {
+      it('should return the displayName of the node', () => {
+        class Foo extends React.Component {
+          render() { return <div />; }
+        }
+
+        Foo.displayName = 'CustomWrapper';
+
+        const wrapper = mount(<Foo />);
+        expect(wrapper.name()).to.equal('CustomWrapper');
+      });
+
+      describeIf(!REACT013, 'stateless function components', () => {
+        it('should return the name of the node', () => {
+          const Foo = () => <div />;
```

Comment:
instead of an arrow (since arrow functions don't have names) could this be a normal function SFC, with a differing name?